frontend-public/src/tests/components/__snapshots__/ReturnPage.spec.js.snap

```diff
@@ -1,78 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ReturnPage renders properly 1`] = `
-ShallowWrapper {
-  Symbol(enzyme.__root__): [Circular],
-  Symbol(enzyme.__unrendered__): <ReturnPage
-    authenticateUser={
-      [MockFunction] {
-        "calls": Array [
-          Array [
-            "blah",
-          ],
-        ],
-      }
-    }
-    location={
-      Object {
-        "search": "type=login&code=blah",
-      }
-    }
-    redirect="/"
-    unAuthenticateUser={[MockFunction]}
```

Comment:
It does look like some of these have been lost. 🤔 There's no longer a reference to `unAuthenticateUser` nor `authenticateUser`, which are provided in the spec as dispatch props.

That may be acceptable, as dispatched actions are probably not coupled tightly enough to the view layer for us to care about them in our snapshots... But it makes me think that there may be no benefit to passing them in when we generate the snapshot. If removing the dispatch function mocks returns a runtime exception during test execution then I'm not sure how to feel.

Thoughts?

Reply:
I think the end result is the point though? The parameters that we pass in the test should result in a redirect to the root - at least as how I understand this test. And the snapshot should capture how that 'Redirect' component is rendered to the client.

If we wanted to capture the various 'outcome renders' that could occur with a component given different properties, then we could create seperate snapshot tests given seperate mocked properties for each case?